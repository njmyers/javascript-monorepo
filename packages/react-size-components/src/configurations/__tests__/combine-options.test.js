import combineOptions from '../combine-options';

describe('it works', () => {
  test('it is a function ', () => {
    expect(typeof combineOptions).toBe('function');
  });

  test('it runs without crashing', () => {
    expect(combineOptions()).toMatchSnapshot();
  });
});
