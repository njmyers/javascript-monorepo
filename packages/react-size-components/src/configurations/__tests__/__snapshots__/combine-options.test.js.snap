// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `
Object {
  "breakpoint": 768,
  "component": false,
  "custom": Array [],
  "inView": false,
  "measureWindow": false,
  "mobile": false,
  "orientation": false,
  "resize": false,
  "scroll": false,
}
`;
